src/features/dailyForecasts/DailyForecastsPage.tsx

```diff
@@ -79,6 +84,7 @@ export const DailyForecastsPage = () => {
           Get Wx Forecast Data
```

Comment:
Can we also change the button label to reflect that it's fetching historic data as well?